package/MDAnalysis/coordinates/PDB.py

```diff
@@ -251,6 +258,8 @@ class PDBReader(base.ReaderBase):
     .. versionchanged:: 2.5.0
        Tempfactors (aka bfactors) are now read into the ts.data dictionary each
        frame.  Occupancies are also read into this dictionary.
+    .. versionchanged:: 2.10.0
+       segID is read from 73-76 instead of 67-76.
```

Comment:
Let's remove this versionchanged from PDBReader because the reader itself does not care, only PDBParser does.

I should have seen this earlier.